MekHQ/resources/mekhq/resources/CampaignGUI.properties

```diff
@@ -3,26 +3,27 @@
 # Menu Resources
 # File Menu
 fileMenu.text = File
-menuLoad.text=Load Campaign...
-menuSave.text=Save Campaign...
+menuLoad.text=Load Campaign
```

Comment:
Why remove the ellipses? That's standard menu speak for "I open a dialog". (I understand for Load and Save in some respects, but the rest?)

Reply:
Late night mistake, I'll revert it once I finish tracking down the medical rating issue.

Reply:
Reverted